modules/etcd-node-scaling.adoc

```diff
@@ -6,11 +6,11 @@
 [id="etcd-node-scaling_{context}"]
 = Node scaling for etcd
 
-In general, clusters must have 3 control plane nodes. However, if your cluster is installed on a bare metal platform, you can scale a cluster up to 5 nodes as a post-installation task. For example, to scale from 3 to 4 nodes after installation, you can add a host and install it as a control plane node. Then, the etcd Operator scales accordingly to account for the additional node.
+In general, clusters must have 3 control plane nodes. However, if your cluster is installed on a bare metal platform, you can scale a cluster up to 5 control plane nodes as a post-installation task. For example, to scale from 3 to 4 control plane nodes after installation, you can add a host and install it as a control plane node. Then, the etcd Operator scales accordingly to account for the additional control plane node.
 
-Scaling to 4-node or 5-node clusters is available only on bare metal platforms.
+Scaling to 4-control-plane-node or 5-control-plane-node clusters is available only on bare metal platforms.
 
-For more information about how to scale from 3 to 4 nodes by using the Assisted Installer, see "Adding hosts with the API" and "Installing a primary control plane node on a healthy cluster".
+For more information about how to scale from 3 to 4 control plane nodes by using the Assisted Installer, see "Adding hosts with the API" and "Installing a primary control plane node on a healthy cluster".
```

Comment:
Since we're touching this, and assuming it applies to 5 control plane nodes, too, how about

```suggestion
For more information about how to scale from 3 to 4 or 5 control plane nodes by using the Assisted Installer, see "Adding hosts with the API" and "Installing a primary control plane node on a healthy cluster".
```